.github/workflows/event-processor.yml

```
@@ -26,17 +30,16 @@ jobs:
       contents: read
     name: Handle ${{ github.event_name }} ${{ github.event.action }} event
```

```suggestion
    name: Handle ${{ github.event_name }} ${{ github.event.action }} event with azure login
```

I know we talked about this but I do think we should at least add something about what is different about this job.